src/Configuration/CFGCommon/CFGArg.json

```diff
@@ -248,7 +265,9 @@
         "To list all connected FPGA devices:",
         "  programmer list_device",
         "To list all connected programming cable:",
-        "  programmer list_cable"
+        "  programmer list_cable",
+        "To configure JTAG frequency for specified cable:",
+        "  programmer jtag_frequency -c <cable_index or cable_name> <frequency in kHz>"
     ]}
   ]
-}
+}
```

Comment:
add empty line

Reply:
the formatting tool is cleaning up empty line. It doesn't allow that